Fix ⚠️ ERROR: No build artifacts found

https://jitpack.io/com/github/hossain-khan/kgeo-timeline/0.1/build.log

```
Build starting...
Start: Tue Mar 26 02:39:48 UTC 2024 c76a036df0d9
Git:
v0.1-0-g0d48377
commit 0d48377337d06fe6c84ddf8c67531d574e8708fe
Merge: e45ab84 2969365
Author: Hossain Khan 
Date:   Mon Mar 25 22:12:30 2024 -0400

    Merge pull request #57 from hossain-khan/feat/29-sample-data

Init SDKMan
Found gradle
Gradle build script
Found gradle version: 8.7.
Using gradle wrapper
Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF-8 -Dhttps.protocols=TLSv1.2
Downloading https://services.gradle.org/distributions/gradle-8.7-bin.zip
.10%.20%.30%.40%.50%.60%.70%.80%.90%.100%

------------------------------------------------------------
Gradle 8.7
------------------------------------------------------------

Build time:   2024-03-22 15:52:46 UTC
Revision:     650af14d7653aa949fce5e886e685efc9cf97c10

Kotlin:       1.9.22
Groovy:       3.0.17
Ant:          Apache Ant(TM) version 1.10.13 compiled on January 4 2023
JVM:          1.8.0_252 (Private Build 25.252-b09)
OS:           Linux 4.10.0-28-generic amd64

0m3.996s
Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF-8 -Dhttps.protocols=TLSv1.2
openjdk version "1.8.0_252"
OpenJDK Runtime Environment (build 1.8.0_252-8u252-b09-1~16.04-b09)
OpenJDK 64-Bit Server VM (build 25.252-b09, mixed mode)
Getting tasks: ./gradlew tasks --all
Tasks: 
Found javadoc task

 ⚠️   WARNING:
 Gradle 'publishToMavenLocal' task not found. Please add the 'maven-publish' or 'maven' plugin.
 See the documentation and examples: https://docs.jitpack.io

Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF-8 -Dhttps.protocols=TLSv1.2
/home/jitpack/build/build.gradle.kts has component: java
/home/jitpack/build/parser/build.gradle.kts has component: java
/home/jitpack/build/sample/build.gradle.kts has component: java
Running: ./gradlew clean -Pgroup=com.github.hossain-khan -Pversion=0.1 -xtest publishToMavenLocal

> Configure project :
w: file:///home/jitpack/build/build.gradle.kts:23:15: The expression is unused
Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF-8 -Dhttps.protocols=TLSv1.2
e: file:///home/jitpack/build/build.gradle.kts:23:13: Expecting an element
e: file:///home/jitpack/build/build.gradle.kts:25:9: Expecting an element
e: file:///home/jitpack/build/build.gradle.kts:32:34: Expecting an element
e: file:///home/jitpack/build/build.gradle.kts:32:46: Expecting an element
e: file:///home/jitpack/build/build.gradle.kts:33:34: Expecting an element
e: file:///home/jitpack/build/build.gradle.kts:42:45: Expecting an element
e: file:///home/jitpack/build/build.gradle.kts:23:1: Function invocation 'apply()' expected
e: file:///home/jitpack/build/build.gradle.kts:23:1: Not enough information to infer type variable T
e: file:///home/jitpack/build/build.gradle.kts:23:7: Unresolved reference. None of the following candidates is applicable because of receiver type mismatch: 
public inline fun ObjectConfigurationAction.plugin(pluginClass: KClass<out Plugin<*>>): ObjectConfigurationAction defined in org.gradle.kotlin.dsl
e: file:///home/jitpack/build/build.gradle.kts:23:15: Too many characters in a character literal ''maven-publish''
e: file:///home/jitpack/build/build.gradle.kts:25:1: Unresolved reference: def
e: file:///home/jitpack/build/build.gradle.kts:28:1: Expression 'publishing' cannot be invoked as a function. The function 'invoke()' is not found
e: file:///home/jitpack/build/build.gradle.kts:28:1: Unresolved reference. None of the following candidates is applicable because of receiver type mismatch: 
public val PluginDependenciesSpec.publishing: PluginDependencySpec defined in org.gradle.kotlin.dsl
e: file:///home/jitpack/build/build.gradle.kts:29:5: Unresolved reference: publications
e: file:///home/jitpack/build/build.gradle.kts:30:9: Expression '"${name}"' of type 'String' cannot be invoked as a function. The function 'invoke()' is not found
e: file:///home/jitpack/build/build.gradle.kts:30:19: Classifier 'MavenPublication' does not have a companion object, and thus must be initialized here
e: file:///home/jitpack/build/build.gradle.kts:32:17: Unresolved reference: def
e: file:///home/jitpack/build/build.gradle.kts:32:54: Unresolved reference: it
e: file:///home/jitpack/build/build.gradle.kts:32:65: Too many characters in a character literal ''java''
e: file:///home/jitpack/build/build.gradle.kts:32:75: Unresolved reference: it
e: file:///home/jitpack/build/build.gradle.kts:32:86: Too many characters in a character literal ''release''
e: file:///home/jitpack/build/build.gradle.kts:33:17: Function invocation 'from(...)' expected
e: file:///home/jitpack/build/build.gradle.kts:33:17: Unresolved reference. None of the following candidates is applicable because of receiver type mismatch: 
public inline fun <reified T : VersionControlSpec> VcsMapping.from(noinline configureAction: TypeVariable(T).() -> Unit): Unit defined in org.gradle.kotlin.dsl
public inline fun <T : VersionControlSpec> VcsMapping.from(type: KClass<TypeVariable(T)>, configureAction: Action<in TypeVariable(T)>): Unit defined in org.gradle.kotlin.dsl
e: file:///home/jitpack/build/build.gradle.kts:35:13: Unresolved reference: groupId
e: file:///home/jitpack/build/build.gradle.kts:36:13: Unresolved reference: artifactId
e: file:///home/jitpack/build/build.gradle.kts:37:23: Unresolved reference: ver
e: file:///home/jitpack/build/build.gradle.kts:42:7: Unresolved reference: publishToMavenLocal

FAILURE: Build completed with 2 failures.

1: Task failed with an exception.
-----------
* Where:
Build file '/home/jitpack/build/build.gradle.kts' line: 23

* What went wrong:
Script compilation errors:

  Line 23: apply plugin: 'maven-publish'
                       ^ Expecting an element

  Line 23: apply plugin: 'maven-publish'
           ^ Function invocation 'apply()' expected

  Line 23: apply plugin: 'maven-publish'
           ^ Not enough information to infer type variable T

  Line 23: apply plugin: 'maven-publish'
                 ^ Unresolved reference. None of the following candidates is applicable because of receiver type mismatch: 
                     public inline fun ObjectConfigurationAction.plugin(pluginClass: KClass<out Plugin<*>>): ObjectConfigurationAction defined in org.gradle.kotlin.dsl

  Line 23: apply plugin: 'maven-publish'
                         ^ Too many characters in a character literal ''maven-publish''

  Line 25: def ver = version
                   ^ Expecting an element

  Line 25: def ver = version
           ^ Unresolved reference: def

  Line 28: publishing {
           ^ Expression 'publishing' cannot be invoked as a function. The function 'invoke()' is not found

  Line 28: publishing {
           ^ Unresolved reference. None of the following candidates is applicable because of receiver type mismatch: 
               public val PluginDependenciesSpec.publishing: PluginDependencySpec defined in org.gradle.kotlin.dsl

  Line 29:     publications {
               ^ Unresolved reference: publications

  Line 30:         "${name}"(MavenPublication) {
                   ^ Expression '"${name}"' of type 'String' cannot be invoked as a function. The function 'invoke()' is not found

  Line 30:         "${name}"(MavenPublication) {
                             ^ Classifier 'MavenPublication' does not have a companion object, and thus must be initialized here

  Line 32:                 def pubComponent = components.find { it.name == 'java' || it.name == 'release' }
                                            ^ Expecting an element

  Line 32:                 def pubComponent = components.find { it.name == 'java' || it.name == 'release' }
                                                        ^ Expecting an element

  Line 32:                 def pubComponent = components.find { it.name == 'java' || it.name == 'release' }
                           ^ Unresolved reference: def

  Line 32:                 def pubComponent = components.find { it.name == 'java' || it.name == 'release' }
                                                                ^ Unresolved reference: it

  Line 32:                 def pubComponent = components.find { it.name == 'java' || it.name == 'release' }
                                                                           ^ Too many characters in a character literal ''java''

  Line 32:                 def pubComponent = components.find { it.name == 'java' || it.name == 'release' }
                                                                                     ^ Unresolved reference: it

  Line 32:                 def pubComponent = components.find { it.name == 'java' || it.name == 'release' }
                                                                                                ^ Too many characters in a character literal ''release''

  Line 33:                 from pubComponent
                                            ^ Expecting an element

  Line 33:                 from pubComponent
                           ^ Function invocation 'from(...)' expected

  Line 33:                 from pubComponent
                           ^ Unresolved reference. None of the following candidates is applicable because of receiver type mismatch: 
                               public inline fun <reified T : VersionControlSpec> VcsMapping.from(noinline configureAction: TypeVariable(T).() -> Unit): Unit defined in org.gradle.kotlin.dsl
                               public inline fun <T : VersionControlSpec> VcsMapping.from(type: KClass<TypeVariable(T)>, configureAction: Action<in TypeVariable(T)>): Unit defined in org.gradle.kotlin.dsl

  Line 35:             groupId = group
                       ^ Unresolved reference: groupId

  Line 36:             artifactId = name
                       ^ Unresolved reference: artifactId

  Line 37:             version = ver
                                 ^ Unresolved reference: ver

  Line 42: tasks.publishToMavenLocal.dependsOn assemble
                                                       ^ Expecting an element

  Line 42: tasks.publishToMavenLocal.dependsOn assemble
                 ^ Unresolved reference: publishToMavenLocal

27 errors

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

2: Task failed with an exception.
-----------
* Where:
Build file '/home/jitpack/build/build.gradle.kts' line: 23

* What went wrong:
Script compilation errors:

  Line 23: apply plugin: 'maven-publish'
                       ^ Expecting an element

  Line 23: apply plugin: 'maven-publish'
           ^ Function invocation 'apply()' expected

  Line 23: apply plugin: 'maven-publish'
           ^ Not enough information to infer type variable T

  Line 23: apply plugin: 'maven-publish'
                 ^ Unresolved reference. None of the following candidates is applicable because of receiver type mismatch: 
                     public inline fun ObjectConfigurationAction.plugin(pluginClass: KClass<out Plugin<*>>): ObjectConfigurationAction defined in org.gradle.kotlin.dsl

  Line 23: apply plugin: 'maven-publish'
                         ^ Too many characters in a character literal ''maven-publish''

  Line 25: def ver = version
                   ^ Expecting an element

  Line 25: def ver = version
           ^ Unresolved reference: def

  Line 28: publishing {
           ^ Expression 'publishing' cannot be invoked as a function. The function 'invoke()' is not found

  Line 28: publishing {
           ^ Unresolved reference. None of the following candidates is applicable because of receiver type mismatch: 
               public val PluginDependenciesSpec.publishing: PluginDependencySpec defined in org.gradle.kotlin.dsl

  Line 29:     publications {
               ^ Unresolved reference: publications

  Line 30:         "${name}"(MavenPublication) {
                   ^ Expression '"${name}"' of type 'String' cannot be invoked as a function. The function 'invoke()' is not found

  Line 30:         "${name}"(MavenPublication) {
                             ^ Classifier 'MavenPublication' does not have a companion object, and thus must be initialized here

  Line 32:                 def pubComponent = components.find { it.name == 'java' || it.name == 'release' }
                                            ^ Expecting an element

  Line 32:                 def pubComponent = components.find { it.name == 'java' || it.name == 'release' }
                                                        ^ Expecting an element

  Line 32:                 def pubComponent = components.find { it.name == 'java' || it.name == 'release' }
                           ^ Unresolved reference: def

  Line 32:                 def pubComponent = components.find { it.name == 'java' || it.name == 'release' }
                                                                ^ Unresolved reference: it

  Line 32:                 def pubComponent = components.find { it.name == 'java' || it.name == 'release' }
                                                                           ^ Too many characters in a character literal ''java''

  Line 32:                 def pubComponent = components.find { it.name == 'java' || it.name == 'release' }
                                                                                     ^ Unresolved reference: it

  Line 32:                 def pubComponent = components.find { it.name == 'java' || it.name == 'release' }
                                                                                                ^ Too many characters in a character literal ''release''

  Line 33:                 from pubComponent
                                            ^ Expecting an element

  Line 33:                 from pubComponent
                           ^ Function invocation 'from(...)' expected

  Line 33:                 from pubComponent
                           ^ Unresolved reference. None of the following candidates is applicable because of receiver type mismatch: 
                               public inline fun <reified T : VersionControlSpec> VcsMapping.from(noinline configureAction: TypeVariable(T).() -> Unit): Unit defined in org.gradle.kotlin.dsl
                               public inline fun <T : VersionControlSpec> VcsMapping.from(type: KClass<TypeVariable(T)>, configureAction: Action<in TypeVariable(T)>): Unit defined in org.gradle.kotlin.dsl

  Line 35:             groupId = group
                       ^ Unresolved reference: groupId

  Line 36:             artifactId = name
                       ^ Unresolved reference: artifactId

  Line 37:             version = ver
                                 ^ Unresolved reference: ver

  Line 42: tasks.publishToMavenLocal.dependsOn assemble
                                                       ^ Expecting an element

  Line 42: tasks.publishToMavenLocal.dependsOn assemble
                 ^ Unresolved reference: publishToMavenLocal

27 errors

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

BUILD FAILED in 1s
Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF-8 -Dhttps.protocols=TLSv1.2
e: file:///home/jitpack/build/build.gradle.kts:23:13: Expecting an element
e: file:///home/jitpack/build/build.gradle.kts:25:9: Expecting an element
e: file:///home/jitpack/build/build.gradle.kts:32:34: Expecting an element
e: file:///home/jitpack/build/build.gradle.kts:32:46: Expecting an element
e: file:///home/jitpack/build/build.gradle.kts:33:34: Expecting an element
e: file:///home/jitpack/build/build.gradle.kts:42:45: Expecting an element
e: file:///home/jitpack/build/build.gradle.kts:23:1: Function invocation 'apply()' expected
e: file:///home/jitpack/build/build.gradle.kts:23:1: Not enough information to infer type variable T
e: file:///home/jitpack/build/build.gradle.kts:23:7: Unresolved reference. None of the following candidates is applicable because of receiver type mismatch: 
public inline fun ObjectConfigurationAction.plugin(pluginClass: KClass<out Plugin<*>>): ObjectConfigurationAction defined in org.gradle.kotlin.dsl
e: file:///home/jitpack/build/build.gradle.kts:23:15: Too many characters in a character literal ''maven-publish''
e: file:///home/jitpack/build/build.gradle.kts:25:1: Unresolved reference: def
e: file:///home/jitpack/build/build.gradle.kts:28:1: Expression 'publishing' cannot be invoked as a function. The function 'invoke()' is not found
e: file:///home/jitpack/build/build.gradle.kts:28:1: Unresolved reference. None of the following candidates is applicable because of receiver type mismatch: 
public val PluginDependenciesSpec.publishing: PluginDependencySpec defined in org.gradle.kotlin.dsl
e: file:///home/jitpack/build/build.gradle.kts:29:5: Unresolved reference: publications
e: file:///home/jitpack/build/build.gradle.kts:30:9: Expression '"${name}"' of type 'String' cannot be invoked as a function. The function 'invoke()' is not found
e: file:///home/jitpack/build/build.gradle.kts:30:19: Classifier 'MavenPublication' does not have a companion object, and thus must be initialized here
e: file:///home/jitpack/build/build.gradle.kts:32:17: Unresolved reference: def
e: file:///home/jitpack/build/build.gradle.kts:32:54: Unresolved reference: it
e: file:///home/jitpack/build/build.gradle.kts:32:65: Too many characters in a character literal ''java''
e: file:///home/jitpack/build/build.gradle.kts:32:75: Unresolved reference: it
e: file:///home/jitpack/build/build.gradle.kts:32:86: Too many characters in a character literal ''release''
e: file:///home/jitpack/build/build.gradle.kts:33:17: Function invocation 'from(...)' expected
e: file:///home/jitpack/build/build.gradle.kts:33:17: Unresolved reference. None of the following candidates is applicable because of receiver type mismatch: 
public inline fun <reified T : VersionControlSpec> VcsMapping.from(noinline configureAction: TypeVariable(T).() -> Unit): Unit defined in org.gradle.kotlin.dsl
public inline fun <T : VersionControlSpec> VcsMapping.from(type: KClass<TypeVariable(T)>, configureAction: Action<in TypeVariable(T)>): Unit defined in org.gradle.kotlin.dsl
e: file:///home/jitpack/build/build.gradle.kts:35:13: Unresolved reference: groupId
e: file:///home/jitpack/build/build.gradle.kts:36:13: Unresolved reference: artifactId
e: file:///home/jitpack/build/build.gradle.kts:37:23: Unresolved reference: ver
e: file:///home/jitpack/build/build.gradle.kts:42:7: Unresolved reference: publishToMavenLocal

FAILURE: Build completed with 2 failures.

1: Task failed with an exception.
-----------
* Where:
Build file '/home/jitpack/build/build.gradle.kts' line: 23

* What went wrong:
Script compilation errors:

  Line 23: apply plugin: 'maven-publish'
                       ^ Expecting an element

  Line 23: apply plugin: 'maven-publish'
           ^ Function invocation 'apply()' expected

  Line 23: apply plugin: 'maven-publish'
           ^ Not enough information to infer type variable T

  Line 23: apply plugin: 'maven-publish'
                 ^ Unresolved reference. None of the following candidates is applicable because of receiver type mismatch: 
                     public inline fun ObjectConfigurationAction.plugin(pluginClass: KClass<out Plugin<*>>): ObjectConfigurationAction defined in org.gradle.kotlin.dsl

  Line 23: apply plugin: 'maven-publish'
                         ^ Too many characters in a character literal ''maven-publish''

  Line 25: def ver = version
                   ^ Expecting an element

  Line 25: def ver = version
           ^ Unresolved reference: def

  Line 28: publishing {
           ^ Expression 'publishing' cannot be invoked as a function. The function 'invoke()' is not found

  Line 28: publishing {
           ^ Unresolved reference. None of the following candidates is applicable because of receiver type mismatch: 
               public val PluginDependenciesSpec.publishing: PluginDependencySpec defined in org.gradle.kotlin.dsl

  Line 29:     publications {
               ^ Unresolved reference: publications

  Line 30:         "${name}"(MavenPublication) {
                   ^ Expression '"${name}"' of type 'String' cannot be invoked as a function. The function 'invoke()' is not found

  Line 30:         "${name}"(MavenPublication) {
                             ^ Classifier 'MavenPublication' does not have a companion object, and thus must be initialized here

  Line 32:                 def pubComponent = components.find { it.name == 'java' || it.name == 'release' }
                                            ^ Expecting an element

  Line 32:                 def pubComponent = components.find { it.name == 'java' || it.name == 'release' }
                                                        ^ Expecting an element

  Line 32:                 def pubComponent = components.find { it.name == 'java' || it.name == 'release' }
                           ^ Unresolved reference: def

  Line 32:                 def pubComponent = components.find { it.name == 'java' || it.name == 'release' }
                                                                ^ Unresolved reference: it

  Line 32:                 def pubComponent = components.find { it.name == 'java' || it.name == 'release' }
                                                                           ^ Too many characters in a character literal ''java''

  Line 32:                 def pubComponent = components.find { it.name == 'java' || it.name == 'release' }
                                                                                     ^ Unresolved reference: it

  Line 32:                 def pubComponent = components.find { it.name == 'java' || it.name == 'release' }
                                                                                                ^ Too many characters in a character literal ''release''

  Line 33:                 from pubComponent
                                            ^ Expecting an element

  Line 33:                 from pubComponent
                           ^ Function invocation 'from(...)' expected

  Line 33:                 from pubComponent
                           ^ Unresolved reference. None of the following candidates is applicable because of receiver type mismatch: 
                               public inline fun <reified T : VersionControlSpec> VcsMapping.from(noinline configureAction: TypeVariable(T).() -> Unit): Unit defined in org.gradle.kotlin.dsl
                               public inline fun <T : VersionControlSpec> VcsMapping.from(type: KClass<TypeVariable(T)>, configureAction: Action<in TypeVariable(T)>): Unit defined in org.gradle.kotlin.dsl

  Line 35:             groupId = group
                       ^ Unresolved reference: groupId

  Line 36:             artifactId = name
                       ^ Unresolved reference: artifactId

  Line 37:             version = ver
                                 ^ Unresolved reference: ver

  Line 42: tasks.publishToMavenLocal.dependsOn assemble
                                                       ^ Expecting an element

  Line 42: tasks.publishToMavenLocal.dependsOn assemble
                 ^ Unresolved reference: publishToMavenLocal

27 errors

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

2: Task failed with an exception.
-----------
* Where:
Build file '/home/jitpack/build/build.gradle.kts' line: 23

* What went wrong:
Script compilation errors:

  Line 23: apply plugin: 'maven-publish'
                       ^ Expecting an element

  Line 23: apply plugin: 'maven-publish'
           ^ Function invocation 'apply()' expected

  Line 23: apply plugin: 'maven-publish'
           ^ Not enough information to infer type variable T

  Line 23: apply plugin: 'maven-publish'
                 ^ Unresolved reference. None of the following candidates is applicable because of receiver type mismatch: 
                     public inline fun ObjectConfigurationAction.plugin(pluginClass: KClass<out Plugin<*>>): ObjectConfigurationAction defined in org.gradle.kotlin.dsl

  Line 23: apply plugin: 'maven-publish'
                         ^ Too many characters in a character literal ''maven-publish''

  Line 25: def ver = version
                   ^ Expecting an element

  Line 25: def ver = version
           ^ Unresolved reference: def

  Line 28: publishing {
           ^ Expression 'publishing' cannot be invoked as a function. The function 'invoke()' is not found

  Line 28: publishing {
           ^ Unresolved reference. None of the following candidates is applicable because of receiver type mismatch: 
               public val PluginDependenciesSpec.publishing: PluginDependencySpec defined in org.gradle.kotlin.dsl

  Line 29:     publications {
               ^ Unresolved reference: publications

  Line 30:         "${name}"(MavenPublication) {
                   ^ Expression '"${name}"' of type 'String' cannot be invoked as a function. The function 'invoke()' is not found

  Line 30:         "${name}"(MavenPublication) {
                             ^ Classifier 'MavenPublication' does not have a companion object, and thus must be initialized here

  Line 32:                 def pubComponent = components.find { it.name == 'java' || it.name == 'release' }
                                            ^ Expecting an element

  Line 32:                 def pubComponent = components.find { it.name == 'java' || it.name == 'release' }
                                                        ^ Expecting an element

  Line 32:                 def pubComponent = components.find { it.name == 'java' || it.name == 'release' }
                           ^ Unresolved reference: def

  Line 32:                 def pubComponent = components.find { it.name == 'java' || it.name == 'release' }
                                                                ^ Unresolved reference: it

  Line 32:                 def pubComponent = components.find { it.name == 'java' || it.name == 'release' }
                                                                           ^ Too many characters in a character literal ''java''

  Line 32:                 def pubComponent = components.find { it.name == 'java' || it.name == 'release' }
                                                                                     ^ Unresolved reference: it

  Line 32:                 def pubComponent = components.find { it.name == 'java' || it.name == 'release' }
                                                                                                ^ Too many characters in a character literal ''release''

  Line 33:                 from pubComponent
                                            ^ Expecting an element

  Line 33:                 from pubComponent
                           ^ Function invocation 'from(...)' expected

  Line 33:                 from pubComponent
                           ^ Unresolved reference. None of the following candidates is applicable because of receiver type mismatch: 
                               public inline fun <reified T : VersionControlSpec> VcsMapping.from(noinline configureAction: TypeVariable(T).() -> Unit): Unit defined in org.gradle.kotlin.dsl
                               public inline fun <T : VersionControlSpec> VcsMapping.from(type: KClass<TypeVariable(T)>, configureAction: Action<in TypeVariable(T)>): Unit defined in org.gradle.kotlin.dsl

  Line 35:             groupId = group
                       ^ Unresolved reference: groupId

  Line 36:             artifactId = name
                       ^ Unresolved reference: artifactId

  Line 37:             version = ver
                                 ^ Unresolved reference: ver

  Line 42: tasks.publishToMavenLocal.dependsOn assemble
                                                       ^ Expecting an element

  Line 42: tasks.publishToMavenLocal.dependsOn assemble
                 ^ Unresolved reference: publishToMavenLocal

27 errors

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
==============================================================================

BUILD FAILED in 1s
Build tool exit code: 0
Looking for artifacts...
Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF-8 -Dhttps.protocols=TLSv1.2
Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF-8 -Dhttps.protocols=TLSv1.2
Looking for pom.xml in build directory and ~/.m2
2024-03-26T02:41:06.023206835Z
Exit code: 0

⚠️ ERROR: No build artifacts found

```

--- a/README.md
+++ b/README.md
@@ -1,3 +1,5 @@
+[![](https://jitpack.io/v/hossain-khan/kgeo-timeline.svg)](https://jitpack.io/#hossain-khan/kgeo-timeline)
+
 # KGeoTimeline 📍
 Kotlin library for parsing Google Location History (Timeline) data.
 

